$NetBSD$

* not to install gamt related files if it is off.

--- GNUmakefile.orig	2008-10-30 09:59:57.000000000 +0000
+++ GNUmakefile
@@ -43,8 +43,10 @@ install: build
 	$(INSTALL_DIR) $(bindir) $(appdir) $(mandir)/man1 $(mandir)/man7
 	$(INSTALL_BINARY) $(TARGETS) $(bindir)
 	$(INSTALL_SCRIPT) amttool $(bindir)
+ifeq ($(HAVE_GTK)$(HAVE_VTE),yesyes)
 	$(INSTALL_DATA) $(DESKTOP) $(appdir)
 	$(INSTALL_DATA) gamt.man $(mandir)/man1/gamt.1
+endif
 	$(INSTALL_DATA) amtterm.man $(mandir)/man1/amtterm.1
 	$(INSTALL_DATA) amttool.man $(mandir)/man1/amttool.1
 	$(INSTALL_DATA) amt-howto.man $(mandir)/man7/amt-howto.7
